examples/webusb/index.js

```diff
@@ -0,0 +1,290 @@
+'use strict';
+/* eslint-env browser */
+
+const assert = require('bsert');
+const bledger = require('../..');
+const {LedgerBcoin} = bledger;
+const {Device} = bledger.WebUSB;
+const {DeviceInfo} = bledger.WebUSB;
+const KeyRing = require('bcoin/lib/primitives/keyring');
+
+const usb = navigator.usb;
```

Comment:
Maybe check to make sure that `navigator` exists first and returning early if it does not